docs/workflows/data/file-write.md

```diff
@@ -0,0 +1,50 @@
+flowclass: Edge 1.0.0
+description: Learn more about the Losant Write: Read Node.
+
+# File Write
+
+The File Write Node allows you to write to a file on your [link](Edge Compute device). This node is *only* available in workflows for your edge device agent.
+
+![File Write Node](/images/workflows/data/file-write-node.png "File Write Node")
+
+## Configuration
+
+There are four configuration settings for this node - file path template, encoding, file content, how to handle writing the file if it exists. All fields expect for encoding are [templatable](/workflows/accessing-payload-data/#string-templates) and can be pulled off of the current workflow payload.
+
+The required fields are file path template, encoding, and file content. The file path template should resolve to the location and file that should be written too. If the file does not exist, we will automatically create it and write the content to that file. By default encoding is `UFT8`. However the following are supported encodings: `ASCII`, `UTF8`, `UFT16LE`, `Base64`, `Binary`, `Hex`. The file content will be the content of the file, or added to the file depending on your selection of `If file exists`.
```

Comment:
"The file path template should resolve to the location and file that should be written too"

"...written to"

Reply:
"The file content will be the content of the file, or added to the file depending on your selection of `If file exists`"

'The file content will be the resolved text of the "File Content Template" field.'